docs/user/user_doc.md

```diff
@@ -53,6 +53,111 @@ poll_url = start_response["url"]
 poll_response = model.poll(poll_url)
 ```
 
+### Uploading Models
+In addition to exploring and running models, the aiXplain SDK allows you to upload your own models to the aiXplain platform. This requires a working model image in line with the template specified [here](https://github.com/aixplain/aiXplain/blob/M-5079254326-container-upload-cli/docs/user/model_setup.md). You will also be required to have an aiXplain account as well as a TEAM_API_KEY which should be set either as an environment variable or passed into each of the following functions.
+
+First, choose a hosting machine appropriate for your model. Note down the host machines "code". You can list the available hosting machines' specifications by running the following:
+```console
+$ aixplain list hosts [--api-key <TEAM_API_KEY>]
```

Comment:
Yes, it can be set in the environment; in fact, that's how I always do it. I put it in brackets because it's always an optional argument, so if you don't want to set an environment variable or you want to use a different TEAM_API_KEY than the one in your own environment, you have the option of passing in a different key.